### DISCUSSION TITLE
wait parameter for "flux create kustomization"

### DISCUSSION THREAD

**Discussion by ruzickap** [fluxcd/flux2#1956] [Proposals] [answered]
Would it be possible to add `wait` parameter (https://fluxcd.io/docs/components/kustomize/kustomization/#health-assessment) to `flux create kustomization` ?

It may be handy for creating "flux kustomization" from the command line...

**Comment by stefanprodan** [comment, marked as answer]
Thanks for this suggestion, implemented in https://github.com/fluxcd/flux2/pull/1961
